resources/references/adr/2025-05-12-implement-measurement-system.md

```diff
@@ -0,0 +1,267 @@
+---
```

Comment:
ADRs will be synced automatically

Reply:
@mitelg do you know when will the ADR be synced?